server.js

```diff
@@ -9,6 +9,11 @@ const sslify = require('express-sslify');
 const debug = require('debug')('cors-proxy:server');
 const morganDebug = require('morgan-debug');
 
+const ORIGIN_WHITELIST = [
+  'tools.taskcluster.net',
+  'status.taskcluster.net'
+];
```

Comment:
For such a simple config, I don't think it is worth adding more complexity to the code, as this will be rarely changed. But I don't any strong opinion on that.